pkg/migration-tool/planner/planner.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 var (
+	second                 = time.Second.Milliseconds()
 	minute                 = time.Minute.Milliseconds()
```

Comment:
should these be const and not var?

Reply:
Sorry, but any functional call over a const (like here time.Second.Milliseconds() over time.Second) is considered as var by the compiler.